proto/extensions.go

```diff
@@ -293,14 +293,22 @@ func ClearExtension(pb Message, extension *ExtensionDesc) {
 
 // GetExtension parses and returns the given extension of pb.
```

Comment:
// GetExtension retrieves a proto2 extended field from pb.
//
// If the descriptor is type complete (i.e., ExtensionDesc.ExtensionType is non-nil),
// then GetExtension parses the encoded field and returns a Go value of the specified type.
// If the field is not present, then the default value is returned (if one is specified),
// otherwise ErrMissingExtension is reported. 
//
// If the descriptor is not type complete (i.e., ExtensionDesc.ExtensionType is nil),
// then GetExtension returns the raw encoded bytes of the field extension.

Reply:
Done